packages/autocomplete-core/src/__tests__/utils.test.ts

```diff
@@ -0,0 +1,15 @@
+import { flatten } from '../utils';
+
+describe('flatten', () => {
+  it('flattens only values', () => {
```

Comment:
What does this test mean? Can we reword that?

Reply:
is 1305d89 better?

Reply:
Yep!